### PR TITLE
Listen page player

### PR DESCRIPTION
**Pull request by mayhem**
# Summary

Implement a listens page player and start the follow feature.

* This is a…
    * [] Bug fix
    * [x] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): 

# Solution

# Action

### CODE REVIEW

File: consul_config.py.ctmpl [hunk 1]
```diff
@@ -68,6 +68,7 @@ INCOMING_EXCHANGE = '''{{template "KEY" "incoming_exchange"}}'''
 INCOMING_QUEUE = '''{{template "KEY" "incoming_queue"}}'''
 UNIQUE_EXCHANGE = '''{{template "KEY" "unique_exchange"}}'''
 UNIQUE_QUEUE = '''{{template "KEY" "unique_queue"}}'''
+FOLLOW_LIST_QUEUE = '''{{template "KEY" "follow_list_queue"}}'''
```
Comment (paramsingh) [inline consul_config.py.ctmpl]: Probably need to add `WEBSOCKETS_SERVER_URL` to this file too.

File: docker/beta/uwsgi/uwsgi.service [hunk 2]
```diff
@@ -34,5 +34,11 @@ then
     exec uwsgi /etc/uwsgi/uwsgi-api-compat.ini
 fi
 
+if [ "${CONTAINER_NAME}" = "listenbrainz-follow-dispatch-beta" ]
+then
+    cd /code/listenbrainz
+    exec python3 manage.py run_follow_server -h 0.0.0.0 -p 8081
```
Comment (paramsingh) [inline docker/beta/uwsgi/uwsgi.service]: I don't think this is the ideal way to run the server in production, uwsgi?

File: listenbrainz/webserver/static/js/profile.js [hunk 3]
```diff
@@ -0,0 +1,687 @@
+'use strict';
```
Comment (paramsingh) [inline listenbrainz/webserver/static/js/profile.js]: we have been using 2 space indentation in our JS and html code, should probably do the same here. (not a biggie, just mentioning)

File: listenbrainz/follow_server/follow_server.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    email_config=app.config.get('LOG_EMAIL'),
+    sentry_config=app.config.get('LOG_SENTRY')
+)
+socketio = SocketIO(app)
+
+
+@socketio.on('json')
+def handle_json(data):
+    current_app.logger.error('received json: %s' % str(data))
+    print('received json: ' + str(data))
```
Comment (mayhem) [inline listenbrainz/follow_server/follow_server.py]: Yuck

File: listenbrainz/webserver/static/js/profile.js [hunk 5]
```diff
@@ -0,0 +1,687 @@
...
+							{this.state.currentSpotifyTrack.duration_ms &&
+								<span className="small"> — {millisecondsToHumanReadable(this.state.currentSpotifyTrack.duration_ms)}</span>
+							}
+						</h3>
+						<p>{this.state.currentSpotifyTrack.artists.map(artist => artist.name).join(', ')}</p>
+						<div>{this.getAlbumArt()}</div>
+					</div>
+				}
+				{this.props.currentListen && this.props.currentListen.user_name &&
+					<div>from {this.props.currentListen.user_name}'s listens</div>
```
Comment (paramsingh) [inline listenbrainz/webserver/static/js/profile.js]: is the red here github's fault? (again, no big issue, just wondering)

File: docker/beta/uwsgi/uwsgi.service (hunk 2 above)
Comment (mayhem) [reply docker/beta/uwsgi/uwsgi.service]: The whole situation of how to run in production is entirely unclear right now. For now, we can let this go, but for production this needs careful consideration. I'm OK to merge as is.

File: consul_config.py.ctmpl (hunk 1 above)
Comment (mayhem) [reply consul_config.py.ctmpl]: I think we can leave this to the production fixes as well.

File: listenbrainz/webserver/static/js/profile.js (hunk 5 above)
Comment (mayhem) [reply listenbrainz/webserver/static/js/profile.js]: No clue.

**Pull request merged**